eslint.config.js

```diff
@@ -4,6 +4,7 @@ import TypescriptParser from '@typescript-eslint/parser'
 import UnicornPlugin from 'eslint-plugin-unicorn'
 import UnusedImportsPlugin from 'eslint-plugin-unused-imports'
 import UnusedClassesPlugin from 'eslint-plugin-tss-unused-classes'
+import NoScrollbarPlugin from 'eslint-plugin-no-scrollbar'
```

Comment:
I don't think this should be a package on npm but you have already published it so anyway.